### ISSUE TITLE
v0.2.42  - connect-redis sessionStore is broken

### ISSUE THREAD

**Issue by ronaldcurtis** [keystonejs/keystone#1019]
I just upgraded keystone using npm to 0.2.42 (from 0.2.22), and I noticed that my connect-redis session store is no longer working, and coming up with a "need express-session" dependency error. I was able to fix this by going to `keystone/lib/core/mount.js` and removing:

```
if (sessionStore === 'connect-redis') {
  var expressSession = require('express-session');
  SessionStore = require(sessionStore)(expressSession);
} else {
  SessionStore = require(sessionStore)(express);
}
```
to just:

```
SessionStore = require(sessionStore)(express);
```
I think this is because keystone (whatever is the latest on npm) is still using Express 3x and does not need express-session yet:
http://stackoverflow.com/questions/22273567/redisstore-typeerror-cannot-read-property-prototype-of-undefined

**Comment by morenoh149** [comment]
is this still an issue for you? could you try the 0.3.0 branch?

**Comment by ronaldcurtis** [comment]
Yes it is fixed on the 0.3.0 branch, just thought to point it out though for it is broken on the stable release.

**Comment by ronaldcurtis** [comment]
Stable release is now 0.3.1. Closing ticket.
